belt-node8-service.gitlab-ci.yml

```diff
@@ -47,7 +47,8 @@ publish:
   script:
     - grunt bundle --packageVersion=$VERSION --packageName=$CI_PROJECT_NAME
   only:
-    - $DEFAULT_BRANCH
+    variables:
+      - $CI_COMMIT_REF_NAME == $DEFAULT_BRANCH
```

Comment:
alternatively, we could change this to check if the branch name is master OR develop

Reply:
Have we tested out what overriding each job would look like? Or yaml inheritance, and overriding just the root?

Wish we'd just standardize our default branch

Reply:
Or we just have our own

Reply:
I could just go through and swap our stuff to master. Seems like we've already spent too much time on this.